R/openapi-spec.R

```diff
@@ -65,11 +65,8 @@ responsesSpecification <- function(endpts){
       if (is.function(endpts$serializer)) {
         ctype <- formals(endpts$serializer)$type
```

Comment:
Could we also skip this check? For plumber serializers, all of these should be `FALSE`.  

And most likely, everyone else is eventually wrapping around `serializer_headers()`

Reply:
sure